datafusion/physical-plan/src/sorts/streaming_merge.rs

```diff
@@ -51,18 +52,34 @@ macro_rules! merge_helper {
     }};
 }
 
-#[derive(Default)]
 pub struct StreamingMergeBuilder<'a> {
     streams: Vec<SendableRecordBatchStream>,
     schema: Option<SchemaRef>,
-    expressions: LexOrderingRef<'a>,
+    expressions: &'a LexOrdering,
     metrics: Option<BaselineMetrics>,
     batch_size: Option<usize>,
     fetch: Option<usize>,
     reservation: Option<MemoryReservation>,
     enable_round_robin_tie_breaker: bool,
 }
 
+static EMPTY_ORDER: OnceLock<LexOrdering> = OnceLock::new();
+
+impl<'a> Default for StreamingMergeBuilder<'a> {
+    fn default() -> Self {
+        Self {
+            streams: vec![],
+            schema: None,
+            expressions: EMPTY_ORDER.get_or_init(LexOrdering::default),
```

Comment:
👍 

What do you think about moving this `EMPTY_ORDER` to `LexOrdering`? 

Something like

```rust
impl LexOrdering {
...
  /// Return an empty LexOrdering (no expressions)
  pub empty() -> &'static LexOrdering { 
    EMPTY_ORDER.get_or_init(LexOrdering::default),
  }
}
```

That would avoid needing to copy LexOrderings quite as much (I'll comment inline